content/blog/pulumi-import-generate-iac-for-existing-cloud-resources/index.md

```diff
@@ -1,7 +1,7 @@
 ---
-title: "Automatically Create Infrastructure Code with Pulumi"
+title: "Pulumi Import: Generate IaC for existing Cloud Resources"
```

Comment:
Consistent title capitalization

"Pulumi Import: Generate IaC for Existing Cloud Resources"